db/migrate/20220406011016_add_sms_notification_preferences_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSmsNotificationPreferencesToUsers < ActiveRecord::Migration[7.0]
```

Comment:
In the long term, I would rather have a notification_preferences table for our long-term durability as a codebase
But I know this was a lot of work- we can save it as a refactoring for someone who wants to practice db migrations

Reply:
sounds good. Maybe open up a new issue?

Reply:
Sure thing, I'll address this in my next pull request.